dex/testing/eth/create-node.sh

```diff
@@ -2,19 +2,18 @@
 # Script for creating eth nodes.
 set -e
 
-# The following are required script arguments
+# The following are required script arguments.
 TMUX_WIN_ID=$1
 NAME=$2
 NODE_PORT=$3
 CHAIN_ADDRESS=$4
 CHAIN_PASSWORD=$5
 CHAIN_ADDRESS_JSON=$6
 CHAIN_ADDRESS_JSON_FILE_NAME=$7
-ADDRESS=$8
-ADDRESS_PASSWORD=$9
-ADDRESS_JSON=${10}
-ADDRESS_JSON_FILE_NAME=${11}
-NODE_KEY=${12}
+ADDRESS_JSON=$8
+ADDRESS_JSON_FILE_NAME=$9
+NODE_KEY=${10}
+SYNC_MODE=${11}
 
 GROUP_DIR="${NODES_ROOT}/${NAME}"
 MINE_JS="${GROUP_DIR}/mine.js"
```

Comment:
I had to add these two lines a few lines below here.

```
mkdir -p "${NODE_DIR}/keystore"
mkdir -p "${NODE_DIR}/geth"
```

Reply:
Added.